app/models/concerns/user_encrypted_attribute_overrides.rb

```diff
@@ -62,17 +64,25 @@ def stale_encrypted_email?
     encrypted_attributes[:email].stale?
   end
 
-  def phone
-    get_encrypted_attribute(name: :phone, default: nil)
+  ENCRYPTED_ATTRIBUTES.each do |attribute|
```

Comment:
Yes, I am familiar with `source_location`. I think in general the expectation is that meta-programming is the exception, not the rule. Totally makes sense in some cases, but not all.

The argument I am putting forth here is that one instance of duplication does not justify meta-programming. Some duplication is OK in my book. It's tempting to reach for meta-programming right away when two methods are the same, but it is sometimes better to wait a bit before abstracting.

Sandi Metz is the one who really hammered this idea into my head https://www.sandimetz.com/blog/2016/1/20/the-wrong-abstraction